src/python/pants/help/help_formatter.py

```diff
@@ -46,19 +46,21 @@ def format_options(self, scope, description, option_registrations_iter):
 
         def add_option(ohis, *, category=None):
             lines.append("")
-            display_scope = scope or "Global"
+            display_scope = f"`{scope}`" or "Global"
             if category:
-                lines.append(self._maybe_green(f"{display_scope} {category}:"))
+                title = f"{display_scope} {category} options"
+                lines.append(self._maybe_green(f"{title}\n{'-' * len(title)}"))
             else:
-                lines.append(self._maybe_green(f"{display_scope}:"))
+                title = f"{display_scope} options"
+                lines.append(self._maybe_green(f"{title}\n{'-' * len(title)}"))
                 if description:
                     lines.append(description)
```

Comment:
A newline above looks like it would bring this display in line with the precedent and read better.

Reply:
Good call.